Automattic/newspack-ads
# feat: activation hook defaults

Implements the following activation defaults, if the settings are not yet found on the database:

 - Default ad units assigned to their respective placements
 - SCAIP with the legacy option turned off (legacy should still be the default option for users that were already using Ads with SCAIP)
 - SCAIP number of repetitions set to 3 in order to use use all 3 in-article default ad units

Closes #411

### How to test

1. On a fresh install, check out this branch
2. Go through the onboarding wizard and activate the Ads plugin
3. Enable GAM and enter a legacy network code
4. Visit the Placements wizard and confirm the active placements have a default unit assigned to them
5. Visit the Add-Ons wizard and enable SCAIP
6. Visit the Settings wizard and confirm legacy mode is off and the number of repetitions is set to 3
7. VIsit the Placements again, edit the SCAIP placements and confirm the default units are assigned to them
